caf_launchpad/main.tf

```diff
@@ -101,6 +109,9 @@ locals {
     prefix             = var.prefix
     prefixes           = var.prefix == "" ? null : [try(random_string.prefix.0.result, var.prefix)]
     prefix_with_hyphen = var.prefix == "" ? null : format("%s", try(random_string.prefix.0.result, var.prefix))
+    suffix             = var.suffix
+    suffixes           = var.suffix == "" ? null : [try(random_string.suffix.0.result, var.suffix)]
+    suffix_with_hyphen = var.suffix == "" ? null : format("%s", try(random_string.suffix.0.result, var.suffix))
```

Comment:
Not sure whether these two could also be useful. Just copied over the idea from the prefix.